modules/adding-tab-pods-page.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-tab-to-pods-page"]
+== Adding a tab to the pods page
```

Comment:
```suggestion
= Adding a tab to the pods page
```
For consistent formatting, we want to keep module titles as H1s, then call the appropriate level with the `include` statement.
See the fourth callout here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Whoops just a typo.